cfme/tests/configure/test_tag.py

```diff
@@ -124,6 +126,29 @@ def test_map_tagging_crud(appliance, category, soft_assert):
                                           .format(map_tag_entity.label))
 
 
+@pytest.fixture
+def csv_tag_file(create_vm, category, tag):
+    temp_file = tempfile.NamedTemporaryFile(suffix=".csv")
```

Comment:
So, you can use tempfiles's context manager here to make this even simpler.
```
with tempfile.NamedTemporaryFile(suffix='.csv') as temp_file:
    tempfile.write(csv_data)
    yield temp_file.name
```

Note this is yielding the file name, not the file object, which is what the test actually needs anyway.

After the yield, the context manager exists and the file is deleted

Reply:
Updated as suggested. I was on the fence about returning the object or just name. I relaize I just use the name, but was thinking I might want to access the object if I used this fixture for another test. Using it in another test seems quite unlikely now.

Reply:
I am getting some errors now when I pass back a string instead of the object. Looking into it. Gong to pass back the object for now so I can push my other changes.

Reply:
getting error trying to use tempfile: TypeError: `expected str, bytes or os.PathLike object, not _TemporaryFileWrapper` . Investigating.

Reply:
I was unable to get the shortened version to work. Tried a few things but was getting errors or creating empty files. Moving on to the asserts for the vm tag.